server/admin_management/api/app/services/application_admin_service.py

```diff
@@ -40,6 +42,9 @@ def create_application_admin(
         fam_user = self.user_service.find_or_create(
             request.user_type_code, request.user_name, request.user_guid, requester
         )
+        fam_user = self.user_service.update_user_properties_from_verified_target_user(
+            fam_user.user_id, target_user, requester
```

Comment:
Hi Ian, I wonder since we'll always add first name, last name and email when create user, should we add those three properties in the find_or_create method? and leave the update business_guid method?

Reply:
Hi Catherine, both could work. 
I feel the "find_or_create" already has complex logic not only conditionally trying to find by "user_name" or "user_guild" and dealing with user_name update, so I feel doing a separate function for a single purpose for just update all other fields (except user_name) is a bit clear to me.

Reply:
right Ian, cause if found user, we should do an update, thanks so much for explanation!!